src/chainparams.cpp

```diff
@@ -265,19 +265,25 @@ class SigNetParams : public CChainParams {
         vSeeds.clear();
 
         if (!args.IsArgSet("-signet_blockscript")) {
-            throw std::runtime_error(strprintf("%s: -signet_blockscript is mandatory for signet networks", __func__));
-        }
-        const auto blockscript = args.GetArgs("-signet_blockscript");
-        if (blockscript.size() != 1) {
-            throw std::runtime_error(strprintf("%s: -signet_blockscript cannot be multiple values.", __func__));
+            LogPrintf("Using default signet network\n");
+            bin = ParseHex("512103ad5e0edad18cb1f0fc0d28a3d4f1f3e445640337489abb10404f2d1e086be430210359ef5021964fe22d6f8e05b2463c9540ce96883fe3b278760f048f5189f2e6c452ae");
```

Comment:
in ad33fd5b5f74bb8ff56ffe901f5a1825e73ecf44

The help info on the args could be updated to reflect the defaults to the global signet. Right now it's not obvious that blockscript and seednode are optional.

Reply:
Otherwise all the changes look good to me!